### DISCUSSION TITLE
IG themes

### DISCUSSION THREAD

**Discussion by DEHNCKA** [d2phap/ImageGlass#1242] [Q&A]
How can I add program theme files to the repository?

**Comment by d2phap** [comment]
for the theme packs, you can submit PR to https://github.com/ImageGlass/theme

**Reply by DEHNCKA** [reply]
> you can submit PR

Okay. What does the abbreviation "PR" mean?

**Reply by DEHNCKA** [reply]
Prerelease?

**Reply by DEHNCKA** [reply]
Pull request?

**Reply by d2phap** [reply]
it's pull request
